### ISSUE TITLE
Report more call stats

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#269]
Our original design had us reporting not just the total number of calls, but the number of calls placed in the last day or week. Now that we're approaching 1,000,000 calls, we should do something like this to give a better sense of call activity.

**Comment by nickoneill** [comment]
And update the total call stats to remove any `test` vias

**Comment by oderby** [comment]
@nickoneill what do you mean by that last comment? Will the /report API now return a breakdown including test calls? I had assumed we'd just discard them in the backend...

**Comment by nickoneill** [comment]
Yeah, we could totally discard as well. I was thinking it wasn't even worth doing since we don't plan on having many test entries overall.

**Comment by dektar** [comment]
(but if you could sanity-check every now and then that there aren't unexpectedly high numbers of via=test coming in, that'd be super...)
